nixos/lib/make-disk-image.nix

```diff
@@ -178,7 +184,7 @@ let format' = format; in let
     nix-store --load-db < ${closureInfo}/registration
 
     echo "running nixos-install..."
-    nixos-install --root $root --no-bootloader --no-root-passwd \
+    nixos-install${suffix} --root $root --no-bootloader --no-root-passwd \
```

Comment:
I think this is a bug. It shouldn't have a suffix & that's not documented anywhere. It must come from how make-derivation overrides name:

https://github.com/NixOS/nixpkgs/blob/master/pkgs/stdenv/generic/make-derivation.nix#L180-L186